src/gbd_mapping_generator/data.py

```diff
@@ -153,24 +153,9 @@ def get_cause_data(with_survey):
                            'yll_only': causes.yll_only.replace({np.NaN: False, 1: True}),
                            'yld_only': causes.yld_only.replace({np.NaN: False, 1: True}),
                            'yll_age_start': causes.yll_age_start.replace({np.NaN: 0}),
-                           'yll_age_end': causes.yll_age_end,
+                           'yll_age_end': causes.yll_age_end.replace({np.NaN: 95}),
```

Comment:
Use 125 instead of 95

Reply:
This causes an error so I left at 95:

```
  File "/home/kjells/repos/gbd_mapping/src/gbd_mapping_generator/data.py", line 202, in get_age_restriction_edge
    _edge = id_map[age_restriction][1]
KeyError: 125.0

```

Reply:
You'll need to update the age restriction edge map.  You're getting back age group 32 when you want age group 235.

Reply:
All the other cause data is using 95. When I replace the NaN value in 
'cutaneous_and_mucocutaneous_leishmaniasis'
with 125 that is not a valid index value in get_age_restriction_edge(). And it doesn't look like it should be...